hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/test/meta/ErrorTransactionMetadataTest.java

```diff
@@ -18,26 +18,25 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
-import com.hedera.node.app.spi.meta.ErrorTransactionMetadata;
+import com.hedera.node.app.spi.meta.TransactionMetadata;
 import com.hederahashgraph.api.proto.java.*;
 import org.junit.jupiter.api.Test;
 
 class ErrorTransactionMetadataTest {
-    final ResponseCodeEnum responseCode = ResponseCodeEnum.INVALID_SIGNATURE;
-    final Throwable throwable = new Throwable("Invalid signature");
-    final TransactionBody txBody = createAccountTransaction();
+    private final ResponseCodeEnum responseCode = ResponseCodeEnum.INVALID_SIGNATURE;
+    private final TransactionBody txBody = createAccountTransaction();
+    private final AccountID payer = AccountID.newBuilder().setAccountNum(42L).build();
```

Comment:
I see `ErrorTransactionMetadata` is deleted. If so, can we delete this test ?

Reply:
I think even though the class was changed, the functionality is still there. I will take a look and merge the different test classes into one.